README.md

```diff
@@ -46,262 +46,20 @@ property `db.url` is set to `db:3306`
 
 ## Development
 
-### Pre-requisites
+First read the relevant OS-specific instructions:
 
-##### Dev Container (recommended for Windows)
+- [Windows](getting-started/windows.md)
+- [Linux](getting-started/linux.md)
+- [MacOS](getting-started/macos.md)
 
-You can run all of IntelliJ inside a docker container, which would allow debugging the code on Windows
-machines. This requires having docker installed and running on your machine.
-To do this, navigate to `.devcontainer/devcontainer.json` in IntelliJ. There should be an icon that pops up next to
-the opening curly brace. Click the icon, then select `Create Dev Container and Clone Sources...`. This should pop up
-a dialog box that allows you to change a few options about the container. Look through them and change what you need,
-then hit the `Build Container and Continue` button. Wait for IntelliJ and Docker to build everything. You may need to
-click a few buttons along the way. Eventually a new IntelliJ window will pop up from the dev container. Follow the
-[Yarn](#nodeyarn) instructions and [Getting Started](#getting-started) steps below with the new window. 
-Use `host.docker.internal` as your db-host argument.
+Then follow the [Getting Started](getting-started/getting-started.md) page to Get Started!
 
-To reopen the container after you've closed it, navigate to the `.devcontainer/devcontainer.json` file again,
-click the icon, and select `Show Dev Containers`. Select the container and it should reopen the second IntelliJ.
-If nothing appears, make sure the docker engine is running (perhaps by opening Docker Desktop).
+### How to Contribute
 
-##### Other Notes for Windows Development
+Here are a few tips for first projects or tasks to learn the autograder:
+- Review the Sequence/Class Diagrams (_coming soon_) and try creating one
+- Add documentation
+- Find a section of under-tested code and add some unit tests
+- Take a look through the GitHub repo's Issues page and find one you like, especially ones labeled "good first issue"
```

Comment:
We can provide a link to a filtered view of the issues page which only displays those with the referenced tag.

Let's add this link over the words "good first issue": 
https://github.com/softwareconstruction240/autograder/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22